api/lib/stores/block-store.js

```diff
@@ -278,4 +325,56 @@ function enrichValidator(validatorInfo, validator) {
   }
 }
 
+function formatNetworkForDB(network) {
+  // removed enabled
+  return {
+    network: {
```

Comment:
why did you choose this nested structure? is the readable in the db?

Reply:
I chose it so I can be able to divide it in three different tables inside the `storeNetwork` function, using `payload[network]` for example

Reply:
I propose removing this function from the PR and using it once to fill the DB with the existing data we have so we don't need to type in every data point.

Reply:
?? This got me kind of lost? How to use it once if we remove it? Ah, you mean moving it in that `networks.js` file you said below to use in development

Reply:
right?

Reply:
no, removing it from the API code and only run it as a script once

Reply:
Wait, but scripts are also part of the API code :sweat_smile: OK, but I see what you mean

Reply:
Not even have the call in the block-store, but have it as something completely independent. This is it, right